x-pack/plugins/infra/public/components/ml/anomaly_detection/flyout_home.tsx

```diff
@@ -291,50 +296,53 @@ const CreateJobTab = (props: CreateJobTab) => {
             }
           />
         </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiCard
-            isDisabled={!props.hasSetupCapabilities}
-            icon={<EuiIcon type={'logoKubernetes'} size="xl" />}
-            title={
-              <FormattedMessage
-                defaultMessage="Kubernetes Pods"
-                id="xpack.infra.ml.anomalyFlyout.create.k8sTitle"
-              />
-            }
-            description={
-              <FormattedMessage
-                defaultMessage="Detect anomalies for memory usage and network traffic on Kubernetes Pods."
-                id="xpack.infra.ml.anomalyFlyout.create.k8sDescription"
-              />
-            }
-            footer={
-              <>
-                {props.hasK8sJobs && (
-                  <EuiButtonEmpty
-                    data-test-subj="infraCreateJobTabRecreateJobsButton"
-                    onClick={props.createK8s}
-                  >
-                    <FormattedMessage
-                      defaultMessage="Recreate jobs"
-                      id="xpack.infra.ml.anomalyFlyout.create.recreateButton"
-                    />
-                  </EuiButtonEmpty>
-                )}
-                {!props.hasK8sJobs && (
-                  <EuiButton
-                    data-test-subj="infraCreateJobTabEnableButton"
-                    onClick={props.createK8s}
-                  >
-                    <FormattedMessage
-                      defaultMessage="Enable"
-                      id="xpack.infra.ml.anomalyFlyout.create.createButton"
-                    />
-                  </EuiButton>
-                )}
-              </>
-            }
-          />
-        </EuiFlexItem>
+        {!props.isHostsPage && (
```

Comment:
Same as above, you could probably add a property to hide the k8s section.

Reply:
Yeah, makes sense I was thinking about it as if in the future there are more asset types it would make more sense, thanks for the suggestion.